ocropus-econf

```diff
@@ -84,14 +84,14 @@ for fname,e,t,m,c in outputs:
 if perfile is not None: perfile.close()
 if allconf is not None: allconf.close()
 
-print "errors    %8d"%errs
-print "missing   %8d"%missing
-print "total     %8d"%total
-print "err       %8.3f %%"%(errs*100.0/total)
-print "errnomiss %8.3f %%"%((errs-missing)*100.0/total)
+print("errors    %8d"%errs)
```

Comment:
Maybe not needed here, but we will add that statement anyway during PR #155 see https://github.com/tmbdev/ocropy/pull/155/files#diff-672c98336ceb05ab25dc90ed9934175f (I hope we can continue on these other PR the next weeks)